src/components/TextInput/styles.js

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components/native';
+import { Colors } from '../../utils/Colors';
+import { Fonts } from '../../utils/Fonts';
```

Comment:
Substituir por Path Alias também.
```suggestion
import { Colors } from '@utils/Colors';
import { Fonts } from '@utils/Fonts';
```